pkg/virt-handler/vm_test.go

```diff
@@ -1696,7 +1696,7 @@ var _ = Describe("VirtualMachineInstance", func() {
 			vmiFeeder.Add(vmi)
 			options := &cmdclient.MigrationOptions{
 				Bandwidth:               resource.MustParse("0Mi"),
-				ProgressTimeout:         150,
+				ProgressTimeout:         300,
```

Comment:
Would it make sense to use the constant?

Reply:
Will do once we agree on a value